scripts/examples/webos_example.sh

```diff
@@ -1,32 +1,19 @@
-#!/usr/bin/env bash
-#
-#    Copyright (c) 2022 Project CHIP Authors
-#
-#    Licensed under the Apache License, Version 2.0 (the "License");
-#    you may not use this file except in compliance with the License.
-#    You may obtain a copy of the License at
-#
-#        http://www.apache.org/licenses/LICENSE-2.0
-#
-#    Unless required by applicable law or agreed to in writing, software
-#    distributed under the License is distributed on an "AS IS" BASIS,
-#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#    See the License for the specific language governing permissions and
-#    limitations under the License.
-#
-
 # Activating connectedhomeip build environment
 source scripts/activate.sh
 
-# Clone webos_sdk
-echo "##### Cloning webOS OSE SDK #####"
-git clone https://github.com/cabin15/webos-ose-ndk third_party/webos_sdk
+# Check if webos_sdk directory exists, clone and extract if it doesn't
+if [ ! -d "third_party/webos_sdk/v2.14.1" ]; then
+    echo "##### Cloning webOS OSE SDK #####"
+    git clone https://github.com/cabin15/webos-ose-ndk third_party/webos_sdk
```

Comment:
This should be a fixed tag to make builds reproducible.

Should this not be a submodule that is pinned to webos?